community_supplied/Chassis/check-re-temperature-ml-hidden-markov.rule

```diff
@@ -0,0 +1,203 @@
+healthbot {
+    topic chassis.temperatures {
```

Comment:
Should this be chassis.temperature (singular instead of plural), to be consistent with other topic names?

Reply:
topic chassis.temperatures is already present, it has default rules.